test/common_framework.py

```diff
@@ -183,18 +183,18 @@ def setUpClassPost(cls):
                     major, minor = release().split('.')[0:2]
                     if not (major == '6' and minor in ['11', '12', '13']):
                         if environ.get("NO_KVM", '0') == '1':
-                            print("\nUsable KVM found but NO_KVM=1 - Disabling\n")
+                            print(" \nUsable KVM found but NO_KVM=1 - Disabling", file=stderr, flush=True)
                         else:
-                            print("\nUsable KVM found\n")
+                            print(" \nUsable KVM found", file=stderr, flush=True)
                             cls.have_kvm = True
                     else:
-                        print("\nUsable KVM found but blacklisted kernel (6.11 - 6.13) - Disabling\n")
+                        print(" \nUsable KVM found but blacklisted kernel (6.11 - 6.13) - Disabling", file=stderr, flush=True)
 
             except FileNotFoundError:
                 pass
             except PermissionError:
                 op = check_output(["getfacl", "/dev/kvm"])
-                print("\nPermissions wrong on /dev/kvm\n%s\n" % op.decode('ASCII'))
+                print(" \nPermissions wrong on /dev/kvm\n%s" % op.decode('ASCII'), file=stderr, flush=True)
```

Comment:
And here?

Reply:
I put a note in the commit message too :-)